### PR TITLE
small fixes

### PR DESCRIPTION
**Pull request by bgruening**
Tests are still missing. Would it be possible to add them?

### CODE REVIEW

File: tools/mesmer/mesmer.xml [hunk 1]
```diff
@@ -1,28 +1,26 @@
-<tool id="mesmer" name="Mesmer" version="@VERSION@.0" profile="17.09">
+<tool id="mesmer" name="Mesmer" version="@VERSION@.0" profile="21.05">
```
Comment (alliecreason) [inline tools/mesmer/mesmer.xml:1]: Recommend 19.01 to be compatible with OHSU galaxy infrastructure.

Comment (bgruening) [reply tools/mesmer/mesmer.xml:1]: Ah, I see. Please note that this version is not supported anymore afaik. If you like I can change it.

Comment (alliecreason) [reply tools/mesmer/mesmer.xml:1]: Understood, yes please change.

Comment (bgruening) [reply tools/mesmer/mesmer.xml]: done

**Pull request merged**